src/app/orders/order-detail/order-detail.page.ts

```diff
@@ -1,15 +1,148 @@
+import { ProfileModel } from 'src/providers/models/profile.model';
 import { Component, OnInit } from '@angular/core';
+import { OrderService } from '../../../providers/services/orders/order.service';
+import { ActivatedRoute, Router } from '@angular/router';
+import { StorageServiceProvider } from '../../../providers/services/storage/storage.service';
+import { CONSTANTS } from '../../../providers/utils/constants';
+import { WidgetUtilService } from '../../../providers/utils/widget';
 
 @Component({
   selector: 'app-order-detail',
   templateUrl: './order-detail.page.html',
   styleUrls: ['./order-detail.page.scss'],
 })
 export class OrderDetailPage implements OnInit {
+  orderItems: any = [];
+  orderDetail: any = {};
+  showImportOrder = false;
+  showLoader = false;
+  showCsvButton = false;
+  showCancelOrder = false;
+  orderItemsAvailable = false;
+  csvData: any[] = [];
+  loaderDownloading: any;
```

Comment:
why it is any define a type to it

Reply:
cvsData or loaderDownloadking?

Reply:
We have not need of cvsData because it is not use in code so I remove it.